src/app.ts

```diff
@@ -1051,6 +1070,55 @@ function registerCgnAPIRoutes(
   );
 }
 
+function registerCgnOperatorSearchAPIRoutes(
+  app: Express,
+  basePath: string,
+  cgnOperatorSearchService: CgnOperatorSearchService,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  bearerSessionTokenAuth: any
+): void {
+  const cgnOperatorController: CgnOperatorSearchController = new CgnOperatorSearchController(
+    cgnOperatorSearchService
+  );
+
+  const cgnMerchantCacheDuration = `${CGN_MERCHANT_CACHE_MAX_AGE_SECONDS} seconds`;
+
+  const cgnMerchantCachingMiddleware = apicache.options({
```

Comment:
There is a reason to have two different cache middleware with two different cache durations with others cached api endpoints?

Reply:
@AleDore could answer to this in a better way, but I think this are different because they are updated at different times.

Reply:
This cache layer works at the instance level (is not shared across the backend instances) and the default values are similar (10 minutes vs 5 minutes). I think that only one cache layer should be enough and we can avoid to add complexity over this. But I leave on you the choose it, if there are other architectural reason for keeping two different cache middlewares.

Reply:
This cache ttl will certainly be increased. The actual 10 minutes are there just as a placeholder, they will be increased very probably.

Reply:
By the way I rename the variable because it's not representing the right name.

Reply:
done https://github.com/pagopa/io-backend/pull/804/commits/7d342ba24b1efeb396af1168d077070de40b0b3e